voc/python/ast.py

```diff
@@ -1894,6 +1917,16 @@ def compare_to(arg):
 
     @node_visitor
     def visit_Call(self, node):
+        # Check if the callable function contains 
+        # not implemented features, if it does, abort
+        # compilation
+        if node.func.id in self.notimplemented:
+            error = self.notimplemented[node.func.id]
+            raise RuntimeError(
```

Comment:
This is raising an error at compile time when the function is called.

We don't really want to crash the compiler: what we want instead is to crash the program that's being compiled in the case the function is called.

Namely, what you want here is to generate opcodes for raising RuntimeError.
You can find examples of that in this same file if you look for `java.THROW`.

Reply:
Oh, ok, looks like I misinterpreted the intended behavior, so, what we want to do is then to replace the behavior of lines with "yield from" to throw an exception? And wouldn't it be better for them to throw a NotImplementedError exception?